File: backend/grafioschtrader-common/src/main/java/grafioschtrader/reportviews/securityaccount/SecurityOpenPositionPerSecurityaccount.java
package grafioschtrader.reportviews.securityaccount;

import java.util.ArrayList;
import java.util.List;

/**
 * Contains all Securityaccounts with the open positions.
 */
public class SecurityOpenPositionPerSecurityaccount {
  public SecurityPositionSummary securityPositionSummary;
  public List<SecurityaccountOpenPositionUnits> securityaccountOpenPositionUnitsList = new ArrayList<>();

}

File: frontend/src/app/entities/view/security.open.position.per.securityaccount.ts
import {SecurityaccountOpenPositionUnits} from './securityaccount.open.position.units';
import {SecurityPositionSummary} from './security.position.summary';

export class SecurityOpenPositionPerSecurityaccount {
  constructor(public securityPositionSummary: SecurityPositionSummary,
              public securityaccountOpenPositionUnitsList: SecurityaccountOpenPositionUnits[]) {
  }
}
